src/test/java/ch/wisv/areafiftylan/OrderRestIntegrationTest.java

```diff
@@ -924,6 +924,43 @@ public void testGetTicketAvailability(){
         then().
             body("ticketType", hasItems(equalTo("EARLY_FULL"), equalTo("REGULAR_FULL")));
     }
+
+    @Test
+    public void testGetOneValidTicket() {
+        insertTestOrders();
+        SessionData login = login("user");
+
+        Ticket ticket = new Ticket(user, TicketType.EARLY_FULL, false, false);
+        ticket.setValid(true);
+        ticketRepository.saveAndFlush(ticket);
+
+        //@formatter:off
+        given().
+            filter(sessionFilter).
+            header(login.getCsrfHeader()).
+        when().
+            get("/users/current/tickets").
+        then().
+            body("object", hasSize(1)).
+            body("[0].owner.username", is("user"));
+        //@formatter:on
+    }
+
+    @Test
+    public void testZeroValidTickets() {
+        insertTestOrders();
+        SessionData login = login("user");
+
+        //@formatter:off
+        given().
+            filter(sessionFilter).
+            header(login.getCsrfHeader()).
+        when().
+            get("/users/current/tickets").
+        then().
+            body("object", hasSize(0));
+        //@formatter:on
+    }
 }
```

Comment:
What about the test when there is no user logged in and the route is queried? Does the API give back the correct result? In the currentUserController I don't see any @PreAuthorize annotation. The team queries have this, but order queries also miss this. I would argue that this is incorrect and should be implemented in the order and ticket routes.